crates/re_viewer/src/ui/view_spatial/scene/scene_part/arrows3d.rs

```diff
@@ -67,9 +67,7 @@ impl Arrows3DPart {
             let origin = glam::Vec3::from(origin);
 
             let radius = radius.map_or(Size::AUTO, |r| Size(r.0));
-            let tip_length = LineStripFlags::get_triangle_cap_tip_length(radius.0);
-            let vector_len = vector.length();
-            let end = origin + vector * ((vector_len - tip_length) / vector_len);
+            let end = origin + vector;
```

Comment:
it would be nice to give arrows a rounded base while we're at it, but then we need something like `CAP_START_EXTEND_OUTWARDS`

Reply:
let's do that then, super easy now :)